@xen-orchestra/backups/_VmBackup.js

```diff
@@ -395,6 +406,24 @@ class VmBackup {
     this._fullVdisRequired = fullVdisRequired
   }
 
+  async _healthCheck() {
+    const settings = this._settings
+
+    if (this._healthCheckSr === undefined) {
+      return
+    }
+
+    // check if current VM has tags
+    const { tags } = this.vm
+    const intersect = settings.healthCheckVmsWithTags.some(t => tags.includes(t)) ?? false
```

Comment:
`Array#some` already returns a boolean.

```suggestion
    const intersect = settings.healthCheckVmsWithTags.some(t => tags.includes(t))
```